src/SFA.DAS.EAS.Portal.Client/Models/IAccountDto.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+using System.Collections.Generic;
+
+namespace SFA.DAS.EAS.Portal.Client.Models
+{
+    public interface IAccountDto
```

Comment:
No need for this interface on the read model.   There is no requirement for the read model to be polymorphic.